doc/api/util.md

```diff
@@ -10,6 +10,45 @@ module developers as well. It can be accessed using:
 const util = require('util');
 ```
 
+## util.callbackify(original)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `original` {Function} An `async` function
+* Returns: {Function} a callback style function
+
+Takes an `async` function (or a function that returns a Promise) and returns a
+function following the common Node.js callback style, i.e. taking a `(err, val)
+=> ...` callback as the last argument. In the callback, the first argument will
+be the rejection reason (or `null` if function call resolved successfully), and
+the second argument will be the resolved value.
+
+For example:
+
+```js
+const util = require('util');
+async function fn() {
+  return await Promise.resolve('hello world');
+}
+const callbackFunction = util.callbackify(fn);
+
+callbackFunction((err, ret) => {
+  if (err) throw err;
+  console.log(ret);
+});
+```
+
+will print something like:
+
+```txt
+hello world
+```
+
+*Note*: Like with most callback style functions, the callback is executed in an
+async context (having a limited stacktrace), and if the callback throws the
```

Comment:
Note that with the debugger attached the stack trace isn't limited (async stack traces). Although I think it's a fair statement here.